crypto/fipsmodule/kem/kem.c

```diff
@@ -0,0 +1,270 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <openssl/base.h>
+
+#include "../../kem/internal.h"
+#include "../delocate.h"
+#include "../ml_kem/ml_kem.h"
+#include "internal.h"
+
+static const uint8_t kOIDMLKEM512[]  = {0xff, 0xff, 0xff, 0xff};
+static const uint8_t kOIDMLKEM768[]  = {0xff, 0xff, 0xff, 0xff};
+static const uint8_t kOIDMLKEM1024[] = {0xff, 0xff, 0xff, 0xff};
```

Comment:
Are there real values for this yet?

Reply:
placeholder values as before. I will change before code freeze.